test/modules/nfts/updateNft.test.ts

```diff
@@ -0,0 +1,60 @@
+import test, { Test } from 'tape';
+import spok, { Specifications } from 'spok';
+import { Nft, MetaplexFile, JsonMetadata } from '@/index';
+import { metaplex, spokSamePubkey } from '../../helpers';
+
+test('it can create and update the nft', async (t: Test) => {
+  // Given we have a Metaplex instance.
+
+  const mx = await metaplex();
+  const imageFile = new MetaplexFile('some_image', 'some-image.jpg');
+  const imageUri = await mx.storage().upload(imageFile);
+
+  // And we uploaded some metadata containing this image.
+  const metadataUri = await mx.storage().uploadJson<JsonMetadata>({
+    name: 'JSON NFT name',
+    description: 'JSON NFT description',
+    image: imageUri,
+  });
+
+  // When we create a new NFT with minimum configuration.
+  const { nft: createdNft } = await mx.nfts().createNft({
```

Comment:
Hey @thlorenz can we maybe just leave the comments for now as visual separator of the tests? The other tests don't use t.comment and I'd like to keep the consistency. We can discuss about refactoring the other tests later but I personally don't think there's much value in using t.comment since it gets drowned in a lot of output in tape. Maybe there's a way to improve that though. But for now let's keep things consistent.

Reply:
@lorisleiva i agree with you, theres so many logs it doesnt really help, should i remove it?

Reply:
Yeah please do, I'll revisit with Thorsten later. Thanks!